convert_state.py

```diff
@@ -1,60 +1,56 @@
+""" a program to convert a state abbreviation to state name or vice-versa """
+import json  # import json for json processing
 
 
-
-import json  # import json for json processing
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """ Create a dictionary of states -> abbreviations,
+    and a dictionary of abbreviations -> states """
+    with open('data.json', 'r') as file:
+        original_data = file.read()
+        read_data = json.loads(original_data)
 
-    print(data)
-    
-    # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
+    state_abbr = read_data  # dictionary of state abbreviations keys and state name values
+    state_abbr2 = {}  # dictionary of state name keys and state abbreviation values
```

Comment:
same with variable above ie. state_key_to_abbrev

Reply:
This name could possibly be a little more descriptive so the comment wouldn't be necessary.

Reply:
good call!
done